apis/placement/v1beta1/stageupdate_types.go

```diff
@@ -147,6 +148,25 @@ func (c *ClusterStagedUpdateRun) SetUpdateRunStatus(status UpdateRunStatus) {
 	c.Status = status
 }
 
+// State represents the desired state of an update run.
+// +enum
+type State string
+
+const (
+	// StateNotStarted indicates the update run has been initialized but execution has not started.
+	// This is the default state after initialization completes.
```

Comment:
The state here is the "desired" state set by the user but the whole description  makes this look like a status instead of spec.